infrastructure/kube/keep-test/relay-maintainer-deployment.yaml

```diff
@@ -23,47 +23,50 @@ spec:
 
       initContainers:
         - name: initcontainer-provision-relay-maintainer
-          image: gcr.io/keep-test-f3e0/initcontainer-provision-relay-maintainer
+          image: gcr.io/keep-test-f3e0/initcontainer-provision-relay-maintainer:latest
           imagePullPolicy: Always
           env:
 
             - name: ETH_RPC_URL
               valueFrom:
-                configMapKeyRef:
-                  name: eth-network-internal
-                  key: rpc-url
+                secretKeyRef:
+                  name: eth-network-ropsten
+                  key: infura-rpc-url
 
             - name: ETH_NETWORK_ID
               valueFrom:
                 configMapKeyRef:
-                  name: eth-network-internal
+                  name: eth-network-ropsten
                   key: network-id
 
             - name: ETH_NETWORK_NAME
-              value: internal
+              value: ropsten
+
+            - name: INFURA_PROJECT_ID
+              value: 414a548bc7434bbfb7a135b694b15aa4
```

Comment:
Do we want to pull this to a secret?

Reply:
Part of me was hoping this would be temporary enough that it wouldn't matter...but you're right, should be done anyway.  I'll move it.